container.go

```diff
@@ -410,8 +411,16 @@ func (c *ContainerRequest) ShouldKeepBuiltImage() bool {
 	return c.FromDockerfile.KeepImage
 }
 
-func (c *ContainerRequest) ShouldPrintBuildLog() bool {
-	return c.FromDockerfile.PrintBuildLog
+func (c *ContainerRequest) BuildLogWriter() io.Writer {
+	if c.FromDockerfile.BuildLogWriter == nil {
+		c.FromDockerfile.BuildLogWriter = os.Stderr
+	}
+
+	if c.FromDockerfile.PrintBuildLog {
```

Comment:
question: having two variable to control feel confusing, thoughts?

Reply:
I answered in the previous question